### PR TITLE
Landing page hero section added

### PR DESCRIPTION
**Pull request by LayanJay**
hero section
mobile view

### CODE REVIEW

**Review by MadhushaPrasad — APPROVED** [review]
done

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]
There is an issue with 768px responsive, Can you please fix it.

**Comment by MadhushaPrasad** [comment]
and push this file also "import Navbar from '../components/Navbar'"

**Review by sithumonline — APPROVED** [review]
Try to clean next-env.d before the marge

File: pages/index.tsx [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+import Navbar from '../components/Navbar'
+
+// modules
+import HeroSection from '../modules/HomePage/hero'
+
+const Homepage = () => {
+  return (
+    <Layout title='Home'>
+      <Navbar />
+      {/* hero section */}
```
Comment (sithumonline) [inline pages/index.tsx]: And this line

**Comment by LayanJay** [comment]
> Try to clean next-env.d before the marge

if I clean this file it will cause the type errors in nextjs components

**Comment by LayanJay** [comment]
> There is an issue with 768px responsive, Can you please fix it.

Fixed.

**Comment by LayanJay** [comment]
> and push this file also "import Navbar from '../components/Navbar'"

Still WIP

**Comment by LayanJay** [comment]
navbar added and I'll add the collapse button animations later (with framer motion)

**Review by chamodshehanka — APPROVED** [review]

**Pull request merged**